lib/transport-utils.js

```diff
@@ -1,542 +1,543 @@
 /* Copyright (c) 2015 Richard Rodger, MIT License */
 /* jshint node:true, asi:true, eqnull:true */
```

Comment:
that will be on the next PR, at this point its still using jshint and mocha.  Was trying to keep it somewhat constrained